### PR TITLE
In zendesk4 connector, we need to know if scheduler start a reload of a project

### PR DESCRIPTION
**Pull request by lukas-slovak**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: src/main/java/com/gooddata/connector/Zendesk4ProcessExecution.java [hunk 1]
```diff
@@ -45,6 +47,14 @@ public void setIncremental(final Boolean incremental) {
         this.incremental = incremental;
     }
 
+    public Boolean getReload() {
+        return reload;
+    }
+
+    public void setReload(final Boolean reload) {
```
Comment (liry) [inline src/main/java/com/gooddata/connector/Zendesk4ProcessExecution.java]: Please add some javadocs describing when the feature will be usable on API.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by liry** [comment]
Plz see https://github.com/gooddata/gooddata-java/blob/master/CONTRIBUTING.md ... issue is missing (and also remove reference to non public issue).

**Comment by lukas-slovak** [comment]
This solves #665.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-java-merge-compliance-check/18/ : SUCCESS in 21s

**Pull request merged**